Src/Microsoft.Dynamic/Utils/ContractUtils.cs

```diff
@@ -49,9 +49,13 @@ public static class ContractUtils {
             }
         }
 
+        public static void RequiresNotNull(object value) {
+            string paramName = nameof(value);
```

Comment:
I don't think this is correct. Doesn't `nameof(value) == "value"`?

Reply:
My bad. I will update.